mailer.http.class.php

```diff
@@ -38,35 +40,60 @@ protected function mailSend($header, $body)
 
     function sparkpost_send()
     {
-        $this->edebug('Preparing request data');
+        $this->debug('Preparing request data');
```

Comment:
it's one of the ugly things i had to do. we can't call `edebug` from templates class